# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/base/content/test/performance/browser_preferences_usage.js
# Commit: dabd7d6836c8
# Full Hash: dabd7d6836c8ed368c06537c3c19a61295d67a18
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2024-06-21 04:13:05
# Regressor Bug: 1728331
# File Overlap Count: 1
# Description:
#   Bug 1728331 - Part 4: Make ContentParent KeepAlives explicit with RAII references, r=smaug,dom-worker-reviewers,asuth
#   
#   This is a fairly significant patch, however it would be difficult to break it
#   down into smaller patches:
#   
# ==============================================================================

diff -r 9e04f49c926e -r dabd7d6836c8 browser/base/content/test/performance/browser_preferences_usage.js
--- a/browser/base/content/test/performance/browser_preferences_usage.js	Thu Jun 20 19:24:51 2024 +0000
+++ b/browser/base/content/test/performance/browser_preferences_usage.js	Thu Jun 20 19:24:51 2024 +0000
@@ -186,8 +186,8 @@
     // This pref is only accessed in automation to speed up tests.
     knownProblematicPrefs["dom.ipc.keepProcessesAlive.webIsolated.perOrigin"] =
       {
-        min: 100,
-        max: 102,
+        min: 50,
+        max: 51,
       };
     if (AppConstants.platform == "linux") {
       // The following sandbox pref is covered by